avocado/utils/disk.py

```diff
@@ -311,3 +311,115 @@ def get_disk_mountpoint(device):
             if dev == device:
                 return fs_dir
         return None
+
+
+def create_linux_raw_partition(disk_name, size=None, num_of_par=1):
+    """
+    Creates partitions using sfdisk command
+
+    :param disk_name: disk/device name
```

Comment:
can we have only disk name as a doc string as device name can confuse the user and even applies to other places as well

Reply:
ok, As suggested will keep only disk name